geode-core/src/test/java/org/apache/geode/internal/process/AbstractProcessStreamReaderIntegrationTest.java

```diff
@@ -149,7 +147,7 @@ protected void givenStartedProcessWithStreamListeners(final Class<?> mainClass)
   }
 
   protected ConditionFactory await() {
-    return Awaitility.await().atMost(WAIT_FOR_READER_IS_RUNNING_TIMEOUT_MILLIS, MILLISECONDS);
+    return Awaitility.await().atMost(2, MINUTES);
```

Comment:
WAIT_FOR_READER_IS_RUNNING_TIMEOUT_MILLIS was poorly named and is now deleted --
 replaced by (2, MINUTES) hardcoded into the await() method that various tests use.

READER_JOIN_TIMEOUT_MILLIS still exists and is used only by the two tests that perform Thread.join style calls on the Readers.

Both timeouts are now 2 minutes which should be more than enough even if the test hits a GC pause.